Source/MQTTnet.Extensions.ManagedClient/ManagedMqttClient.cs

```diff
@@ -42,15 +42,15 @@ public class ManagedMqttClient : IManagedMqttClient
         private Task _maintainConnectionTask;
 
         private ManagedMqttClientStorageManager _storageManager;
-
-        private bool _disposed;
-
+        
         public ManagedMqttClient(IMqttClient mqttClient, IMqttNetChildLogger logger)
         {
             _mqttClient = mqttClient ?? throw new ArgumentNullException(nameof(mqttClient));
 
             if (logger == null) throw new ArgumentNullException(nameof(logger));
             _logger = logger.CreateChildLogger(nameof(ManagedMqttClient));
+
+            Options = new ManagedMqttClientOptions();
```

Comment:
@chkr1011 im not sure about this one. I tried to test managed client to reproduce #828 but while starting MQTTnet.TestApp.NetCore i got nullpointer exptions because options where null. the testcode publishes messages before starting the managed client so 2 options initialize the options with a default in the ctor or throw an invalid operation exption during publish if the client was not started, i went with the first option